FixedPopupPanels/FixedPopupPanels.js

```diff
@@ -53,11 +54,13 @@ const fixedPopupPanelsHandlers = {
  * @ui
  * @public
  */
-const FixedPopupPanels = (props) => {
-	const handlers = useHandlers(fixedPopupPanelsHandlers, props);
-
-	return <FixedPopupPanelsBase {...props} {...handlers} />;
-};
+const FixedPopupPanels = I18nContextDecorator(
+	{rtlProp: 'rtl'},
+	(props) => {
+		const handlers = props.rtl ? null : useHandlers(fixedPopupPanelsHandlers, props);
```

Comment:
Instead of this, let's use `forProp` function.

Reply:
It's fixed.